assessments/README.md

```diff
@@ -2,45 +2,50 @@
+current security related state, background and considerations, it's stated agenda for security, 
+and it's communication patterns and security practices.  
+The following describe high level goals and motivations the security assessment:
 
 ### 1) Reduce risk across the ecosystem
 
 The primary goal is to reduce the risk from malicious attacks and accidental
-breaches of privacy. This process supports that goal in two ways:
+breaches of privacy. This goal is enabled by the security assessment in two ways:
 
+   * The discussion around the security assessment brings security to the mindset of project maintainers
+     who may have otherwise focused on functional achievements
    * Clear and consistent process for communication increases detection &
-     reduces time to resolve known or suspected vulnerability issues
-   * A collaborative evaluation process increases domain expertise within each
-     participating project.
+     reduces time to resolve known or suspected security issues
+   * A collaborative critical process led by industry experts increases domain expertise within
+     participating projects
+   * The assessment gives a measurable security baseline for future reviews
 
-### 2) Accelerate adoption of cloud native technologies
+### 2) Accelerate adoption of Cloud Native technologies
 
-Security reviews are a necessary, time intensive process. Each company,
-organization and project must perform its own reviews to ensure that it meets
-its unique commitments to its own users and stakeholders. In open source, simply
-finding security-related information can be overwhelmingly difficult and a time
-consuming part of the security review. The CNCF security assessment, hereafter
-"security assessment," process is intended to enable improved discovery of
-security information & assist in streamlining internal and external security
-reviews in multiple ways:
+The security assessment yields a thoughtful document authored by the project maintainers and reviewed 
+by security assessment members. This document enables improved discovery of security information & assists 
+in streamlining internal and external security reviews in multiple ways:
 
    * Consistent documentation reduces review time
    * Established baseline of security-relevant information reduces Q&A
-   * Clear rubric for security profile enables organizations to align their risk
-     profile with the project’s risk profile and effectively allocate resources
-     (for review and needed project contribution)
-   * Structured metadata allows for navigation, grouping and cross-linking
+   * Clear rubric for security profile enables potential users to align their risk
+     profile with the project’s risk profile
 
-We expect that this process will raise awareness of how specific open source
-projects affect the security of a cloud native system; however, separate
-activities may be needed to achieve that purpose using materials generated by
-the assessements.
```

Comment:
This paragraph felt devoid for me, but please let me know if I was missing something important there